src/Components/Editors/EasyMDE.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-declare(strict_types=1);
```

Comment:
Why were all of these removed? Can you please add them back?

Reply:
Sorry, by mistake.